membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -174,25 +164,24 @@ object AttributesFromZuora extends LoggingWithLogstashFields {
     futureResult
   }
 
-  def dynamoAndZuoraAgree(maybeDynamoAttributes: Option[Attributes], maybeZuoraAttributes: Option[Attributes], identityId: String): Boolean = {
-    val zuoraAttributesWithIgnoredFields = maybeZuoraAttributes map { zuoraAttributes =>
-      maybeDynamoAttributes match {
-        case Some(dynamoAttributes) => zuoraAttributes.copy(
-          AdFree = dynamoAttributes.AdFree, //fetched from Dynamo in the Zuora lookup anyway (dynamo is the source of truth)
-          MembershipNumber = dynamoAttributes.MembershipNumber, //I don't think membership number is needed and it comes from Salesforce
-          MembershipJoinDate = dynamoAttributes.MembershipJoinDate.flatMap(_ => zuoraAttributes.MembershipJoinDate), //only compare if dynamo has value
-          TTLTimestamp = dynamoAttributes.TTLTimestamp //TTL only in dynamo
-        )
-        case None => zuoraAttributes
-      }
+  def dynamoAndZuoraAgree(maybeDynamoAttributes: Option[DynamoAttributes], maybeZuoraAttributes: Option[ZuoraAttributes], identityId: String): Boolean = {
+    val dynamoAttributesForComparing: Option[ZuoraAttributes] = maybeDynamoAttributes map { dynamoAttributes =>
```

Comment:
pedantic review comment: dynamoAttributesForComparison?

Reply:
Sure, that is better